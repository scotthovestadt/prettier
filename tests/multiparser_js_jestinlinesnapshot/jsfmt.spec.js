jest.spyOn(require("jest"), "getVersion").mockImplementation(() => "24.6.0");

run_spec(__dirname, ["babel", "flow", "typescript"], { proseWrap: "always" });
