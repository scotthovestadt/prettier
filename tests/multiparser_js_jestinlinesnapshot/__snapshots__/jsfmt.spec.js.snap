// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`jestinlinesnapshot-multi-line.js 1`] = `
====================================options=====================================
parsers: ["babel"]
printWidth: 80
proseWrap: "always"
                                                                                | printWidth
=====================================input======================================
it("is a test", () => {
  expect(tree).toMatchInlineSnapshot(\`
Object {
  "createdAt": 2018-05-19T23:36:09.816Z,
  "id": 3,
  "name": "LeBron James",
}
\`);

  expect(nextState).toMatchInlineSnapshot(\`
    Object {
      "createdAt": 2018-05-19T23:36:09.816Z,
      "id": 3,
      "name": "LeBron James",
    }
  \`);

  expect(tree).toMatchInlineSnapshot(\`
          Object {
            "createdAt": 2018-05-19T23:36:09.816Z,
            "id": 3,
            "name": "LeBron James",
          }
        \`);

        expect(tree).toMatchInlineSnapshot(\`
        Object {
          "createdAt": 2018-05-19T23:36:09.816Z,
          "id": 3,
          "name": "LeBron James",
        }
      \`);
});
=====================================output=====================================
it("is a test", () => {
  expect(tree).toMatchInlineSnapshot(\`
    Object {
      "createdAt": 2018-05-19T23:36:09.816Z,
      "id": 3,
      "name": "LeBron James",
    }
  \`);

  expect(nextState).toMatchInlineSnapshot(\`
    Object {
      "createdAt": 2018-05-19T23:36:09.816Z,
      "id": 3,
      "name": "LeBron James",
    }
  \`);

  expect(tree).toMatchInlineSnapshot(\`
    Object {
      "createdAt": 2018-05-19T23:36:09.816Z,
      "id": 3,
      "name": "LeBron James",
    }
  \`);

  expect(tree).toMatchInlineSnapshot(\`
    Object {
      "createdAt": 2018-05-19T23:36:09.816Z,
      "id": 3,
      "name": "LeBron James",
    }
  \`);
});

================================================================================
`;

exports[`jestinlinesnapshot-single-line.js 1`] = `
====================================options=====================================
parsers: ["babel"]
printWidth: 80
proseWrap: "always"
                                                                                | printWidth
=====================================input======================================
it("is a test", () => {
  expect(tree).toMatchInlineSnapshot(\`<Hello />\`);
});

=====================================output=====================================
it("is a test", () => {
  expect(tree).toMatchInlineSnapshot(\`<Hello />\`);
});

================================================================================
`;
